### PR TITLE
Added failing integration test

### PR DESCRIPTION
**Pull request by alexsaves**
I don't expect you to merge this PR - but its just to illustrate a strange corner case I'm seeing with dates generated by Google Sheets. See attached bug. #1002

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Reverse engineered from Excel spreadsheet files as a project.
 # Installation
 
 ```shell
-npm install exceljs
+npm install
```
[the remark this answers is not in the document]
Comment (alexsaves) [reply README.md]: Yes - done

**Comment by guyonroche** [comment]
@alexsaves with the updates, the integration tests pass! See: [https://travis-ci.org/exceljs/exceljs/builds/604768305?utm_source=github_status&utm_medium=notification](https://travis-ci.org/exceljs/exceljs/builds/604768305?utm_source=github_status&utm_medium=notification)

**Comment by alexsaves** [comment]
> @alexsaves with the updates, the integration tests pass! See: https://travis-ci.org/exceljs/exceljs/builds/604768305?utm_source=github_status&utm_medium=notification

I see that it is - but (and this is going to sound crazy) it's still failing for me! Could I be dealing with a locale issue? I've merged your master into mine, done a fresh pull, reinstalled.. and I'm getting: 

https://drive.google.com/open?id=1_OUXvH0p8wQVE27Uyz45nqEbP0piHeWP

```
1) WorkbookReader
       with a spreadsheet that contains dates
         with a cell that contains a date value early in the year
           should decode the month properly as 2:

      AssertionError: expected 1 to equal 2
      + expected - actual
```
In a completely separate test application also - if I bring in the 3.3.1 dependency from, using a different spreadsheet - I also still see this issue. Would you mind humoring me and pulling my branch locally and running the tests?

node ver 12.13.0

**Comment by guyonroche** [comment]
@alexsaves - it is most likely a time-zone issue. I tried your branch but like with Travis, it passes for me, but then I'm in London where the time zone is GMT (aka UTC). As mentioned - the date value stored in the spreadsheet is 42795.0, the '.0' part implies that Excel has recognised it is a date value (not date-time) and is storing it without the time fraction so that it can display the same date in all time zones.
In your test, you successfully extract a Date() value of "2017-03-01T00:00:00.000Z" but the getFullYear(), getMonth() and getDate() functions will be converting this date-time into your local time-zone (at a guess - you're west of UTC which will give you the previous date).
I expect if you switch to using getUTCFullYear(), getUTCMonth() and getUTCDate(), the test will start passing for you too.

**Comment by alexsaves** [comment]
This was indeed the source of my problem. I won't speculate on any improvements to exceljs for this case - but I'm not sure I would have gotten this without your help. Thanks very much!

**Comment by guyonroche** [comment]
@alexsaves no prob, it's always worth getting to the bottom of the problem :-)
I'll close this PR then
